app/views/provisioned.html

```diff
@@ -0,0 +1,156 @@
+<project-header class="top-header"></project-header>
+<project-page>
+
+  <!-- Middle section -->
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header header-toolbar">
+        <div class="container-fluid">
+          <div class="page-header page-header-bleed-right page-header-bleed-left">
+            <h1>
+              Provisioned Services
+              <span class="page-header-link">
+                <a ng-href="{{'services' | helpLink}}" target="_blank">
+                  Learn More <i class="fa fa-external-link" aria-hidden="true"></i>
+                </a>
+              </span>
+            </h1>
+          </div>
+          <div class="data-toolbar">
+            <div class="data-toolbar-filter">
+              <project-filter></project-filter>
+            </div>
+          </div>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content">
+        <div class="container-fluid">
+          <alerts alerts="alerts"></alerts>
+          <div class="row">
+            <div class="col-md-12">
+              <table class="table table-bordered table-hover table-mobile table-layout-fixed">
+                <colgroup>
+                  <col class="col-sm-3">
+                </colgroup>
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th>Bindings</th>
```

Comment:
Seems like you should use the overview-service-binding component. Might need a modification or three to be able to show the small number of differences between showing it for an application or a service, but otherwise there is a lot of overlap.